frontend/src/modules/contributor/pages/contributor-details.page.vue

```diff
@@ -92,20 +92,19 @@ import LfIcon from '@/ui-kit/icon/Icon.vue';
 import LfContributorDetailsHeader from '@/modules/contributor/components/details/contributor-details-header.vue';
 import LfContributorDetailsActions from '@/modules/contributor/components/details/contributor-details-actions.vue';
 import { useRoute } from 'vue-router';
-import { ContributorApiService } from '@/modules/contributor/services/contributor.api.service';
-import { useLfSegmentsStore } from '@/modules/lf/segments/store';
 import { storeToRefs } from 'pinia';
 import LfSpinner from '@/ui-kit/spinner/Spinner.vue';
 import LfContributorDetailsEmails from '@/modules/contributor/components/details/contributor-details-emails.vue';
-import { Contributor } from '@/modules/contributor/types/Contributor';
 import LfContributorLastEnrichment from '@/modules/contributor/components/shared/contributor-last-enrichment.vue';
 import { useMemberStore } from '@/modules/member/store/pinia';
-
-const lsSegmentsStore = useLfSegmentsStore();
-const { selectedProjectGroup } = storeToRefs(lsSegmentsStore);
+import { useContributorStore } from '@/modules/contributor/store/contributor.store';
 
 const { getMemberCustomAttributes } = useMemberStore();
 
+const contributorStore = useContributorStore();
+const { getContributor } = contributorStore;
+const { contributor } = storeToRefs(contributorStore);
```

Comment:
Do we need to keep the contributor in store?

Reply:
we have this approach in organizations and its best to also have it here as then i can use just update method from store and it will update the contributor and i dont need to pass reload back to the page component